net/dnsviz/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# $Id$
+
+PortSystem         1.0
+
+PortGroup          github 1.0
+PortGroup          python 1.0
+
+github.setup       dnsviz dnsviz 0.6.5 v
+github.tarball_from releases
+
+categories         net
+platforms          darwin
+license            GPL-2+
+supported_archs    noarch
+maintainers        deccio.net:casey
+
+description        DNS measurement and diagnostic tool suite
+long_description \
+  DNSViz is a tool suite for measurement, diagnostic, and visualization of \
```

Comment:
Sorry. That's all line numbers, I just wanted to continue my nitpicking where I first started it. You may try to count the spaces (all characters before the "second column") and you'll see there are 19 of them everywhere except for `github.tarball_from releases` which is correct. My editor would automatically align tabs at multiples of 4.